src/transformers/models/llama/configuration_llama.py

```diff
@@ -60,7 +60,7 @@ class LlamaConfig(PretrainedConfig):
             `num_attention_heads`.
         hidden_act (`str` or `function`, *optional*, defaults to `"silu"`):
             The non-linear activation function (function or string) in the decoder.
-        max_position_embeddings (`int`, *optional*, defaults to 2048):
+        max_position_embeddings (`int`, *optional*, defaults to 4096):
```

Comment:
The default value in the config file shouldn't change, as this isn't backwards compatible. Somebody initializing a new model from scratch would not get the same model with these changes: 

```py
from transformers import LlamaConfig, LlamaModel

config = LlamaConfig
# Different model is created
model = LlamaModel(config)
```

Reply:
this IMO should still be adressed